lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -215,7 +215,7 @@ class window.Turbolinks
         else
           refreshedNodes.push(newNode)
 
-      else if existingNode.getAttribute("refresh-always") == null
+      else if existingNode.getAttribute("data-tg-refresh-always") == null && existingNode.getAttribute("refresh-always") == null
```

Comment:
Can we make the attribute methods global and use them here as well?

Bonus to use it in query selectors as well to generate the attribute selector